README.md

```diff
@@ -324,6 +324,13 @@ Emitted when a LevelDB query is made using the `leveldown` module.
     * `value` (Object) The value being added to the LevelDB database using the `put` method (Undefined for other methods) 
     * `opCount` (Number) The number of operations carried out by a `batch` method (Undefined for other methods) 
     * `duration` (Number) the time taken for the LevelDB query to be responded to in ms.
+    
+### Event: 'loopback-datasource-juggler'
+Emitted when a function is called on the `loopback-datasource-juggler` module
```

Comment:
What kind of functions is this intercepting? Any function defined in `loopback-datasource-juggler`? Or only static & prototype methods on `Model` classes? I don't mind either, it's just something that's not clear to me from this description.